examples/app/store/index.js

```diff
@@ -0,0 +1,23 @@
+import {cmsReducer} from "@datawheel/canon-cms";
+import {createLogger} from "redux-logger";
+
+/** */
+export const initialState = {};
+
+/** */
+export const middleware = [];
+
+if (__DEV__ && !__SERVER__) {
+  const loggerMiddleware = createLogger({
+    collapsed: (getState, action, logEntry) => !logEntry.error
+  });
+  middleware.push(loggerMiddleware);
+}
+
+/**
+ * The object exported by this file should contain reducers to be
+ * combined with the internal default canon reducers.
+ */
+export const reducers = {
+  cms: cmsReducer
+};
```

Comment:
I really like reconfiguring all of the redux stuff into a `/store` directory. Can you write up migration steps for anyone updating their version of canon? (maybe just leave the write-up in this PR's main comment, and I'll copy/paste it into the release notes when ready)

Reply:
the migration is pretty simple:

- Create `store/` folder inside `app/` on the application, and create a `index.js` file inside
- Move the default exported object from `app/reducers/index.jsx` to this file, use `export const reducers =`
- Move the default exported object from `app/store.js` to this file, use `export const initialState =` (notice case)
- If used, move the middleware from `canon.js` to this file. In canon.js the middleware was enabled with a function `(applyMiddleware, canonMiddleware) => StoreEnhancer`, where the user had to add the middleware needed to a list of core middleware, and return `applyMiddleware(...array of middlewares)`. The new way only needs to export a `const middleware = []` as an array of the user defined middleware; the internals will do the combination with the core middleware and the call to `applyMiddleware`. The example also shows a way to add a middleware only while running on development mode.
- Update the references to needed imported objects
- Delete the old `app/reducers/index.jsx` and `app/store.js` files.